src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+
+import java.io.*;
+
 public class WorkWithFile {
+    int buySum = 0;
+    int supplySum = 0;
```

Comment:
The variables 'buySum' and 'supplySum' should be local variables within the 'readFromFile' method since they are only used there and do not represent the state of the object.